src/nvim/window.c

```diff
@@ -59,6 +59,7 @@
 #include "nvim/move.h"
 #include "nvim/normal.h"
 #include "nvim/option.h"
+#include "nvim/option_defs.h"
```

Comment:
Why did you add this header? I don't see why your change should need that?

Reply:
Ahh, that should not be in there. Whoops. This is a leftover when it was still "clamp to -cmdheight, not -1", and later became "just clamp"

Reply:
(I suspect it's left over from when you tried to use `cmdheight` instead of a static `1`.)